runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -406,8 +403,8 @@ MM_ScavengerDelegate::hasIndirectReferentsInNewSpace(MM_EnvironmentStandard *env
 	/* Iterate though Class Statics and Constant Dynamics*/
```

Comment:
this comment should be changed to more generic

Reply:
done